devenv-generate/src/main.rs

```diff
@@ -90,6 +91,26 @@ async fn main() -> Result<()> {
 
     log::init_tracing(level, cli.log_format);
 
+    // Show one-time telemetry warning if not disabled and not shown before
+    if !cli.disable_telemetry && !has_shown_telemetry_warning() {
+        println!("\nGenerating collects anonymous usage data sent to https://devenv.new (see privacy policy) to improve generation output.");
```

Comment:
From the Privacy policy linked in the footer of the website, it is not really clear to me what data will be collected.